### DISCUSSION TITLE
Welcome to motrix-webextension Discussions!

### DISCUSSION THREAD

**Discussion by gautamkrishnar** [gautamkrishnar/motrix-webextension#66] [Announcements]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by modees** [comment]
Extencion not working on Arc Browser! Can you update the extension and add browser support?

https://arc.net/



![The-Arc-Browser-in-Windows (Medium)](https://github.com/user-attachments/assets/262e1a16-1b36-459f-91be-dfc272c937af)

**Comment by Clay-dpv** [comment]
我知道如何使用,但是怎么安装呢?没办法把他直接作用到浏览器中

**Reply by gautamkrishnar** [reply]
Just go to chrome webstore to install it https://chromewebstore.google.com/detail/motrix-webextension/djlkbfdlljbachafjmfomhaciglnmkgj
